src/server/server_family.cc

```diff
@@ -52,6 +53,18 @@ extern "C" {
 
 using namespace std;
 
+struct ReplicaOfFlag {
+  string host;
+  string port;
```

Comment:
Would it make sense to make `port` of an integer type, like maybe `uint16_t`?

Reply:
Not really, we supply arguments to `ReplicaOfInternal` via `CmdArgList`, which accepts strings.